dev-packages/cli/src/run-test.ts

```diff
@@ -36,15 +36,26 @@ export default async function runTest(options: TestOptions): Promise<void> {
         matchAppUrl: () => true, // all urls are application urls
         newPage: async () => {
             const browser = await puppeteer.launch(launch);
-            return browser.newPage();
+            // re-use empty tab
+            return new Promise(resolve => {
+                browser.pages().then(p => resolve(p[0]));
+            });
```

Comment:
```suggestion
            const [tab] = await browser.pages();
            return tab;
```

Reply:
Much cleaner - thanks! I have made the change in the corresponding commit, adding a note in the commit message to credit the fact that I have included a suggested change from you.